### ISSUE TITLE
cannot output on non-FilterableStream

### ISSUE THREAD

**Issue by Ifaliuk** [u2takey/ffmpeg-go#93]
I have some code:
<code>
ffmpeg.Input(inFileName).Output(outFileName, ffmpeg.KwArgs{
	"acodec": "pcm_s16le",
	"af":     "afftdn",
	"ar":     16000,
}).OverWriteOutput().WithOutput(os.Stdout).Run()
</code>		
it is raise panic: cannot output on non-FilterableStream
but when I run compiled command in terminal: 
ffmpeg -i ./676376294.mp3 -acodec pcm_s16le -af afftdn -ar 16000 ./676376294_out.wav -y
it's working fine

**Comment by flyingsnoopy** [comment]
Were you also get the `0xc00000a020 error: %s (0x7ff672ccf7a0,0xc0001231a0) exit status 1`?

**Comment by Ifaliuk** [comment]
> Did you also get the `0xc00000a020 error: %s (0x7ff672ccf7a0,0xc0001231a0) exit status 1`?

No, only panic with message: cannot output on non-FilterableStream

**Comment by flyingsnoopy** [comment]
Anyway,maybe you can try to update your ffmpeg.
